services/meta/handler.go

```diff
@@ -322,25 +339,11 @@ func (h *handler) serveLease(w http.ResponseWriter, r *http.Request) {
 			scheme = "https://"
 		}
 
-		leader = scheme + leader + "/lease"
+		leader = scheme + leader + "/lease?name=" + name + "&nodeid=" + nodeIDStr
```

Comment:
There may be other context in the URL parameters from the requester that we don't need to validate, or other values added at a later point. So, I think it would be preferable to do:

```go
leader = scheme + leader + "/lease? + q.Encode()
```